libtransmission/announcer-common.h

```diff
@@ -157,10 +157,10 @@ struct tr_announce_response
  * This is only an upper bound: if the tracker complains about
  * length, announcer will incrementally lower the batch size.
  */
-auto inline constexpr TR_MULTISCRAPE_MAX = 60;
+auto inline constexpr MultiscrapeMax = 60;
 
-auto inline constexpr TR_ANNOUNCE_TIMEOUT_SEC = std::chrono::seconds{ 45 };
-auto inline constexpr TR_SCRAPE_TIMEOUT_SEC = std::chrono::seconds{ 30 };
+auto inline constexpr AnnounceTimeoutSec = std::chrono::seconds{ 45 };
+auto inline constexpr ScrapeTimeoutSec = std::chrono::seconds{ 30 };
```

Comment:
_nit_ slightly awkward to put something in the global namespace without some kind of a `tr` prefix.

Since this is a private header, arguably we could namespace the entire header? I'd prefer that we either do that, or revert the name change here

Reply:
My intention here was just to fix a clang-tidy warning about the naming convention. I've added back the prefix in 
7773483.

Reply:
Eventually the http and udp announce modules should be unified under a single interface class, and these global variables should be moved inside.

It's going to be a challenging refactor though.